Realm/ObjectStore/impl/apple/external_commit_helper.cpp

```diff
@@ -95,10 +95,20 @@ ExternalCommitHelper::ExternalCommitHelper(RealmCoordinator& parent)
     }
 
 #if !TARGET_OS_TV
-    auto path = parent.get_path() + ".note";
+    auto dir_path = parent.get_path() + ".management/";
+    auto path = dir_path + "notification_pipe.note";
```

Comment:
does this need an extension? Weird to call it "note{...}.note". Why not just `notification_pipe`?